uber/templates/jobs/staffers.html

```diff
@@ -67,6 +69,26 @@
 </table>
 
 <br/> <br/>
+
+{% if c.HAS_ACCOUNTS_ACCESS %}
+    <div>
+    <button class="btn btn-default" onclick="checkAll()">Select All</button>
+    <button class="btn btn-default" onclick="uncheckAll()">Deselect All</button>
+    <h4>Accounts Selected: <i id="count">0</i></h4>
+    <form id="admin_settings" method="post" action="add_bulk_admin_accounts" onsubmit="return add_admin_list()" role="form" class="form-inline">
+        <input type="hidden" name="ids" id="ids" value="" />
+        {% csrf_token %}
+        <div class="form-group">
+            {% checkgroup AdminAccount.access %}
+        </div>
+        <button type="submit" class="btn btn-default">Create Checked Accounts</button>
```

Comment:
I might add a confirmation like "This will create {X} admin accounts. Are you sure you want to do this?"